api4/license.go

```diff
@@ -154,6 +177,17 @@ func requestTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	canStart, err := c.App.Srv().LicenseManager.CanStartTrial()
```

Comment:
```suggestion
	canStartTrialLicense, err := c.App.Srv().LicenseManager.CanStartTrial()
```

Reply:
Done.

Reply:
I think this one was forgotten@harshilsharma63